coderd/database/databasefake/databasefake.go

```diff
@@ -775,6 +775,156 @@ func (q *fakeQuerier) GetAuthorizedWorkspaces(ctx context.Context, arg database.
 	return workspaces, nil
 }
 
+func (q *fakeQuerier) GetWorkspaceCount(ctx context.Context, arg database.GetWorkspaceCountParams) (int64, error) {
+	count, err := q.GetAuthorizedWorkspaceCount(ctx, arg, nil)
+	return count, err
+}
+
+//nolint:gocyclo
+func (q *fakeQuerier) GetAuthorizedWorkspaceCount(ctx context.Context, arg database.GetWorkspaceCountParams, authorizedFilter rbac.AuthorizeFilter) (int64, error) {
+	q.mutex.RLock()
+	defer q.mutex.RUnlock()
+
+	workspaces := make([]database.Workspace, 0)
```

Comment:
You could probably save some code by just converting this from `database.GetWorkspaceCountParams` to `database.GetWorkspaceParams` and then calling `q.GetAuthorizedWorkspace` but completely optional.

Reply:
I agree with this!